docs/Concepts/Merge.md

```diff
@@ -0,0 +1,84 @@
+---
+description: What is the Merge?
+---
+
+# The Merge
+
+The Ethereum upgrade known as [The Merge](https://ethereum.org/en/upgrades/merge/) will merge the [Beacon Chain] into
+Ethereum Mainnet, turning Mainnet into a combination of an
+[execution layer and consensus layer](#execution-and-consensus-clients).
+The merge will transition Mainnet from proof of work to [proof of stake consensus](#merge-consensus).
+
+## Execution and consensus clients
+
+After The Merge, a full Ethereum Mainnet node will be a combination of an execution client (sometimes called an
```

Comment:
Can we swap out "sometimes called" for "previously calle"? We should recognize the old names while encouraging use of the newer terminology.

Reply:
done